test/functional/android/appium_tests.py

```diff
@@ -131,7 +131,7 @@ def test_pull_folder(self):
     def test_background_app(self):
         self.driver.background_app(1)
         sleep(5)
-        el = self.driver.find_element_by_name('Animation')
+        el = self.driver.find_element_by_android_uiautomator('new UiSelector().text("Animation")')
```

Comment:
I'd rather use non-uiautomator locator as possible since the locator does not work on Espresso driver.
Can we use another `find_element_by_x`?

Reply:
Yeah, I understand. Updated the tests again with non-uiautomator locator strategy. Thanks for the info!

Reply:
thanks :)